docs/source/getting-started.md

```diff
@@ -50,13 +52,13 @@ model = MyModel() # The PyTorch model to export
 example_inputs = (torch.randn(1,3,64,64),) # A tuple of inputs
 
 et_program =
```

Comment:
formatting still looks a bit off on the website

https://docs-preview.pytorch.org/pytorch/executorch/8618/getting-started.html

Reply:
https://github.com/pytorch/executorch/pull/8637